### DISCUSSION TITLE
modifiers on taxonomy term listing

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#7090] [Troubleshooting]
How do I modify a taxonomy term list, let's say I have:
```antlers
{{ taxonomy:product_collection }}
    {{ title }} 
{{ /taxonomy:product_collection }}
```
and want to display it as a sentence list for example. adding ` | sentence_list` to title or `taxonomy:product_collection` doesn't work.

**Comment by jasonvarga** [comment]
```antlers
{{ {taxonomy:product_collection} | pluck:title | sentence_list }}
```

**Comment by fredbouin** [comment]
thanks @jasonvarga,

What I am actually doing is a bit more complex. 

I'm looping through the `product_collection` taxonomy getting the entries tagged with each one and then listing what `type` taxonomy each of those entries is tagged with.

```antlers
{{ taxonomy:product_collection as="expand" }}
    {{ expand }}
        {{ entries }}
            {{ type }}
                {{ title }}
            {{ /type }}
        {{ /entries }}
    {{ /expand }}
{{ /taxonomy:product_collection }}
```
I thought the simple version I posted before would give me the tools I need to work on this but I'm not figuring out the syntax for this scenario. 

I would think something like this maybe.
```antlers
{{ taxonomy:product_collection as="expand" }}
    {{ expand }}
        {{ entries }}
            {{ type | pluck:title | sentence_list }}
        {{ /entries }}
    {{ /expand }}
{{ /taxonomy:product_collection }}
```
but that does not work.

**Reply by jasonvarga** [reply]
That code seems to work perfectly here. Are you using the runtime parser?

**Reply by fredbouin** [reply]
I'm getting:
Statamic\Modifiers\ModifierException
Call to undefined method Statamic\Taxonomies\LocalizedTerm::map()

And I believe I am running runtime, in antlers.php I have 
```php
'version' => 'runtime',
```

**Reply by fredbouin** [reply]
So I'm able to do:
```antlers
{{ taxonomy:product_collection  }}
  {{ entries | pluck:price | sentence_list }}
{{ /taxonomy:product_collection }}
```
price just being a simple text field
by that logic I might do 
```antlers
{{ taxonomy:product_collection  }}
  {{ entries | pluck:type | sentence_list }}
{{ /taxonomy:product_collection }}
```
but type is a taxonomy which it doesn't seem to like since this doesn't work either.
